src/main/java/com/byznass/tiolktrack/kernel/crypto/SaltGenerator.java

```diff
@@ -0,0 +1,27 @@
+package com.byznass.tiolktrack.kernel.crypto;
+
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class SaltGenerator {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(TokenEncrypter.class);
+
+	public byte[] generateSalt() throws TiolkTrackException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)